### ISSUE TITLE
flyway library is not working with hibernate enver

### ISSUE THREAD

**Issue by manishgithub1173** [flyway/flyway#1957]
###### Which version and edition of Flyway are you using?
latest

###### If this is not the latest version, can you reproduce the issue with the latest one as well?
(Many bugs are fixed in newer releases and upgrading will often resolve the issue)
yes

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)
gradle

###### Which database are you using (type & version)?
mysql 5

###### Which operating system are you using?
MacOS

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, the SQL statement that failed (if relevant) and the command you ran.)

###### What did you expect to see?
Audit tables should have been created.

###### What did you see instead?
Audit tables not created.

Basically, i am using enver library to create audit table for each entity. But if i integrate flyway and enver library, it gives error as it is not allowing to create revinfo table and other audit tables automatically. 
How can i make flyway work with hibernate enver? Is it possible?
